turtlebot3c-bringup-snap/README.md

```diff
@@ -21,7 +21,7 @@ it only starts the `robot_state_publisher` since the robot is meant to be spawne
 ## How to install
```

Comment:
In the instructions above it stills mention the robot_state_publisher. Is it on purpose?

Reply:
That is still true tho, I mean the robot_state_publisher is the only thing that will stay alive with simulation set to True